core/src/main/java/org/elasticsearch/action/fieldstats/FieldStats.java

```diff
@@ -216,7 +261,16 @@ public final void accumulate(FieldStats other) {
         isAggregatable |= other.isAggregatable;
 
         assert type == other.getType();
-        updateMinMax((T) other.minValue, (T) other.maxValue);
+        if (other.hasMinMax == false) {
+            return;
+        }
+        if (hasMinMax == false) {
+            hasMinMax = true;
+            minValue = (T) other.minValue;
+            maxValue = (T) other.maxValue;
```

Comment:
I pushed https://github.com/elastic/elasticsearch/pull/21821/commits/be820240a27ce57e0fd825728b3dab11276b0a6d

Though I think we should distinguish null values because the shard is empty and null values because the field is not indexed. I could not do it in this commit because SeqNoFieldType#stats computes min/max values on a non-searchable field.
@bleskes @jasontedor what is the plan for this SeqNoFieldType#stats ? Are we going to index this field and remove the min/max retrieval ?